molecule/securedrop-docker/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: grsecurity-build-securedrop-trusty
+    image: ubuntu:xenial
```

Comment:
Nit: platform should match image. Tricky use case, though, since we're using Xenial _for_ Trusty, at least until we migrate to Xenial full time. Won't block merge, but flagging it for notice since it may confuse maintainers in the interim.